pyproject.toml

```diff
@@ -1,23 +1,21 @@
 [tool.poetry]
-name = "haicosystemDemo"
+name = "socialstream"
 version = "0.0.1"
 description = "An demo"
-authors = ["Xuhui Zhou <zhouxuhui2018@gmail.com>"]
+authors = ["Zhe Su <suzhesz00@gmail.com>"]
```

Comment:
also include my name :)

Reply:
Oh I see. I was assuming there could be only one author!